pkg/sdk/logging/model/output/vmware_loginsight.go

```diff
@@ -0,0 +1,188 @@
+// Copyright © 2024 Kube logging authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package output
+
+import (
+	"github.com/cisco-open/operator-tools/pkg/secret"
+
+	"github.com/kube-logging/logging-operator/pkg/sdk/logging/model/types"
+)
+
+// +name:"VMware LogInsight"
+// +weight:"200"
+type _hugoVMwareLogInsight interface{} //nolint:deadcode,unused
+
+// +docName:"VMware LogInsight output plugin for Fluentd"
+/*
+For details, see [https://github.com/vmware/fluent-plugin-vmware-loginsight](https://github.com/vmware/fluent-plugin-vmware-loginsight).
+
+## Example output configurations
+
+```yaml
+spec:
+  vmwareLogInsight:
+    scheme: https
+    ssl_verify: true
+    host: MY_LOGINSIGHT_HOST
+    port: 9543
+    agent_id: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+    log_text_keys:
+	- log
+	- msg
+	- message
+    http_conn_debug: false
+```
+*/
+type _docVMwareLogInsight interface{} //nolint:deadcode,unused
+
+// +name:"VMware LogInsight"
+// +url:"https://github.com/vmware/fluent-plugin-vmware-loginsight/releases/tag/v1.4.2"
+// +version:"1.4.2"
+// +description:"Store logs in VMware LogInsight"
+// +status:"GA"
+type _metaVMwareLogInsight interface{} //nolint:deadcode,unused
+
+// +kubebuilder:object:generate=true
+// +docName:"VMwareLogInsight"
+// LogZ Send your logs to VNMware LogInsight
```

Comment:
Is this a copy-paste issue perhaps?
```suggestion
// Send your logs to VNMware LogInsight
```

Reply:
ah, yes. i will get that fixed 👍

Reply:
ok, this is fixed now. thank you!